modules/claims_api/app/workers/claims_api/poa_updater.rb

```diff
@@ -7,24 +7,28 @@ module ClaimsApi
   class PoaUpdater
     include Sidekiq::Worker
 
-    def perform(power_of_attorney_id)
+    def perform(power_of_attorney_id) # rubocop:disable Metrics/MethodLength
       poa_form = ClaimsApi::PowerOfAttorney.find(power_of_attorney_id)
       service = BGS::Services.new(
         external_uid: poa_form.external_uid,
         external_key: poa_form.external_key
       )
+
+      ssn = poa_form.auth_headers['va_eauth_pnid']
+      file_number = service.people.find_by_ssn(ssn)[:file_nbr] # rubocop:disable Rails/DynamicFindBy
+
       response = service.vet_record.update_birls_record(
-        ssn: poa_form.auth_headers['va_eauth_pnid'],
+        file_number: file_number,
+        ssn: ssn,
         poa_code: poa_form.form_data['serviceOrganization']['poaCode']
       )
 
       if response[:return_code] == 'BMOD0001'
         poa_form.status = ClaimsApi::PowerOfAttorney::UPDATED
+        # Clear out the error message if there were previous failures
+        poa_form.vbms_error_message = nil if poa_form.vbms_error_message.present?
 
-        if enable_vbms_access?(poa_form: poa_form)
-          veteran_participant_id = poa_form.auth_headers['va_eauth_pid']
-          ClaimsApi::VBMSUpdater.perform_async(poa_form.id, veteran_participant_id)
-        end
+        ClaimsApi::VBMSUpdater.perform_async(poa_form.id) if enable_vbms_access?(poa_form: poa_form)
```

Comment:
This change loses the 2nd arg (`veteran_participant_id`) to the `VBMSUpdater` job.
I believe any calls to `ClaimsApi::VBMSUpdater.perform_async()` in the specs should be blowing up because of this.

Reply:
[fixed](https://github.com/department-of-veterans-affairs/vets-api/pull/9322/commits/d944c17824f019fa4d0dea8252974f1325c3331f)